### PR TITLE
feat: allow saving of different system config values during startup

### PR DESCRIPTION
**Pull request by jochenmanz**
Screenshot 2023-02-22 at 22 41 36
 It now stops when a error occours :)

### CODE REVIEW

File: devenv.nix [hunk 1]
```diff
@@ -56,83 +56,53 @@ let
...
 
   scriptUpdateConfig = pkgs.writeScript "scriptUpdateConfig" ''
+    VENDOR=${config.env.DEVENV_ROOT}/vendor/autoload.php
+    CONSOLE=${config.env.DEVENV_ROOT}/bin/console
+
+    echo "Updating system config"
+
+    if [ ! -f "$VENDOR" ] || [ ! -f "$CONSOLE" ];
+    then
+      echo "Vendor folder or console command not found. Please run composer install."
```
Comment (tinect) [inline devenv.nix:82]:
```suggestion
      echo "Vendor folder or console not found. Please run composer install."
```

This is just a check if the file exists, isn't it?

File: devenv.nix [hunk 2]
```diff
@@ -56,83 +56,53 @@ let
...
+      exit 1
+    fi
+
     # additional config
     ${lib.concatMapStrings ({ name, value }: ''
-      ${updateConfig} ${name} "${lib.escapeShellArg value}"
+      $CONSOLE system:config:set ${name} ${value} || exit 1
```
Comment (tinect) [inline devenv.nix:88]: There should be a specific message?

**Review by tinect — DISMISSED** [review]

File: devenv.nix (hunk 1 above)
Comment (jochenmanz) [reply devenv.nix:82]: true :D

File: devenv.nix (hunk 2 above)
Comment (jochenmanz) [reply devenv.nix:88]: Errors are always displayed in devenv :)

**Pull request merged**